qiskit/transpiler/target.py

```diff
@@ -1025,7 +1025,7 @@ def target_to_backend_properties(target: Target):
             for qargs, props in qargs_list.items():
                 property_list = []
                 if props is not None:
-                    if props.duration is not None:
+                    if getattr(props, "duration", None) is not None:
```

Comment:
Was there a reason you changed this (and the similar changes below)? It seems generally unrelated to the change in this PR. 

Also I thought we guarded against this with the `if props is None` above, were you hitting a case where the properties were something besides `InstructionProperties` objects?

Reply:
yeah I guess the guard exists here, but it did not exist several lines down (L1061 and L1070). So I unified the style.
Actually I kind of like removing the guard and making the indentation level less. So I just did that here. Let me know if you don't like that.